app/src/assets/localization/en/protocol_command_text.json

```
@@ -31,14 +31,17 @@
   "engaging_magnetic_module": "Engaging Magnetic Module",
   "fixed_trash": "Fixed Trash",
   "home_gantry": "Homing all gantry, pipette, and plunger axes",
+  "in": "in",
```

```suggestion
  "in_location": "in {{location}}",
```
I don't think we can hardcode a string literal to handle i18n, because in some languages, the preposition may proceed the direct object.